lib/utils/asciiToTrytes.js

```diff
@@ -71,7 +71,7 @@ function fromTrytes(inputTrytes) {
     if ( typeof inputTrytes !== 'string' ) return null
 
     // If input length is odd, return null
-    if ( inputTrytes.length % 2 ) return null
+    if ( inputTrytes.length % 2 ) inputTrytes = normalizeTrytes(inputTrytes)
```

Comment:
I don't think we need to outsource this into a separate method.
Also, 'normalize' is a misleading label - you're not normalizing the trytes themselves.